OneCommunityGlobal/HGNRest
# Papia creates suggestion icon

# Description
(PRIORITY HIGH) Jae: Add “Suggestions” Icon to Dashboard icons section
It should function just like the Report App Bug icon (see below) but have the option to be submitted anonymously. 
Should have a dropdown that includes these options:
Suggestion Category: 
To identify and remedy poor client and/or user service experiences
To identify bright spots and enhance positive service experiences
To make fundamental changes to our programs and/or operations
To inform the development of new programs/projects
To identify where we are less inclusive or equitable across demographic groups
To strengthen relationships with the people we serve
To understand people's needs and how we can help them achieve their goals
Other
Would you like a followup/reply regarding this feedback? 
Should include an edit button for an Owner to add to the above “Suggestion Category:” dropdown
Should include an edit button for an Owner to add an additional information collection field if needed

## Related PRS (if any):
To test this backend PR you need to checkout the #998 frontend PR.

## Main changes explained:
created a json data file for suggestion modal data, added get and post function for  adding value to suggestion data file and fetching the data from frontend, added create suggestion mail body function and send suggestion email function

## How to test:
1. check into current branch in backend and frontend
2. follow  this link https://docs.google.com/document/d/1j92U7Vr14gQLehr1e_gkMdqWcOA37WGxNLv9yTa96CY/edit and Aishwarya Kalkundrikar's comment on how to send email
3. do 'npm run build' and 'npm start' to run this PR locally
4. rest, follow frontend PR

## Note:
I will suggest go to dashboardController.js-> sendMakeSuggestion function -> emailSender -> change the email address from oneCommunity to your gmail. So that you can easily verify that email successfully send.
